net/haproxy/src/opnsense/mvc/app/views/OPNsense/HAProxy/index.volt

```diff
@@ -308,50 +361,155 @@ POSSIBILITY OF SUCH DAMAGE.
+            <br/>
+        </div>
+    </div>
+
+    <div id="subtab_haproxy-virtual-services-introduction" class="tab-pane fade">
+        <div class="col-md-12">
+            <h1>Virtual Services</h1>
+            <p>{{ lang._("HAProxy requires two virtual services for it's loadbalancing and proxying features. The following virtual services must be configured for everything that should be served by HAProxy:") }}</p>
+            <ul>
+              <li>{{ lang._('%sBackend Pools:%s The HAProxy backend. Group the %spreviously added servers%s to build a server farm. All servers in a group usually deliver the same content. The Backend Pool cares for health monitoring and load distribution. A Backend Pool must also be configured if you only have a single server. The same Backend Pool may be used for multiple Public Services.') | format('<b>', '</b>', '<b>', '</b>') }}</li>
+              <li>{{ lang._('%sPublic Services:%s The HAProxy frontend. The Public Service listens for client connections, optionally applies rules and forwards client request data to the selected Backend Pool for load balancing (or proxying). Every Public Service needs to be connected to a %spreviously created Backend Pool%s.') | format('<b>', '</b>', '<b>', '</b>') }}</li>
+            </ul>
+            <p>{{ lang._('Remember to add firewall rules for all configured Public Services.') }}</p>
+            <br/>
+        </div>
+    </div>
+
+    <div id="subtab_haproxy-rules-checks-introduction" class="tab-pane fade">
+        <div class="col-md-12">
+            <h1>Rules &amp; Checks</h1>
+            <p>{{ lang._("These are all optional features, HAProxy can be used without them. After getting used to HAProxy you'll find these optional features pretty useful:") }}</p>
+            <ul>
+              <li>{{ lang._('%sHealth Monitors:%s The HAProxy "health checks". Health Monitors are used by %sBackend Pools%s to determine if a server is still able to respond to client requests. If a server fails a health check it will automatically be removed from a Backend Pool and healthy servers are automatically re-added.') | format('<b>', '</b>', '<b>', '</b>') }}</li>
+              <li>{{ lang._('%sConditions:%s HAProxy is capable of extracting data from requests, responses and other connection data and match it against predefined patterns. Use these powerful patterns to compose a condition that may be used in (multiple) Rules.') | format('<b>', '</b>') }}</li>
+              <li>{{ lang._('%sRules:%s Perform a large set of actions if one or more %sConditions%s match. These Rules may be used in %sBackend Pools%s as well as %sPublic Services%s.') | format('<b>', '</b>', '<b>', '</b>', '<b>', '</b>', '<b>', '</b>') }}</li>
```

Comment:
The markup is, err, extensive... it may kill the reading flow.

Reply:
Well, these were supposed to be links in the first place, but I couldn't figure out how to properly set the anchor links ;)